admin/admin.go

```diff
@@ -97,6 +99,8 @@ func New(ctx context.Context, opts *Options, logger *zap.Logger, issuer *auth.Is
 		issuer:           issuer,
 		VersionNumber:    opts.VersionNumber,
 		metricsProjectID: metricsProjectID,
+		// TODO: Validate the syntax, if err or missing, then default to "CRON_TZ=America/Los_Angeles 0 0 * * 1"
```

Comment:
Can remove this TODO – see other comment about setting a default.

As for validation, the cron library will do it when it is invoked and return the error, so we don't need to do additional validation here IMO.

Reply:
Agree!